src/com/kimlew/javarenamefiles/RenameFiles.java

```diff
@@ -85,15 +85,17 @@ private static void getDirectoryFilesAndDisplay(File[] filesOfFileTypeArray) {
     System.out.println(theList);
   }
 
-  private static boolean doesDirectoryExist(File file) {
+  private static boolean directoryDoesNotExist(File file) {
     // Check File object to see if directory exists.
+    boolean directoryDoesNotExist = file.exists();
```

Comment:
You should call this local variable `directoryExists`, since that is more consistent with `file.exists()` returning `true`.

Reply:
@kcharter Thanks. I will change it so it makes more sense.